### PR TITLE
Project cleanup

### LINKED ISSUE #42: Task: Restructure folder structure for backend/frontend

### PR DESCRIPTION
**Pull request by emilie-martin**
Big project cleanup! closes #42 
* uniformized the indentation, import order and general code formatting
* separated pages from components
* transitioned most of the `.css` files to `.ts` files to use `useStyles` with Material UI
* broke `orderBike` into components:
    * `BikeBilling` takes care of the billing
    * `BikeComponents` takes care of displaying the bike components to be selected by the user
    * `BikeModel` takes care of displayed the selected bike parts
 > the same logic breakdown was applied to for orderComponent

Functionality remains untouched and tests all pass.

### CODE REVIEW

**Review by Manraj45 — CHANGES_REQUESTED** [review]
The "i" should be capital. In your app.tsx and UserLogs.tsx, your import has a capital while your file is not.
image
image

**Review by Manraj45 — CHANGES_REQUESTED** [review]
Registration is broken. @nasaku898 had this issue and fixed it by removing the apostrophe from the recovery question. For example: What is your pet's name? should be: What is your pet name?
image

**Comment by emilie-martin** [comment]
> The "i" should be capital. In your app.tsx and UserLogs.tsx, your import has a capital while your file is not.
> image
> image

While I was making those changes I noticed that error pop up but I was fairly sure I had fixed it because it all runs fine on my end? Should I just change it manually from Git? Because it looks fine on my end and the UserLogs component works fine on my end.
image

Also, I'm not too sure why we're trying to get the `InventoryStyles` . I'll change it to ` UserLogsStyles`

**Comment by emilie-martin** [comment]
> Registration is broken. @nasaku898 had this issue and fixed it by removing the apostrophe from the recovery question. For example: What is your pet's name? should be: What is your pet name?
> image

Oh I see. I had changed it so that it was proper English but I'll revert it. Must have forgotten to double-check the registration after changing it; that was my bad.

**Comment by Manraj45** [comment]
> Also, I'm not too sure why we're trying to get the `InventoryStyles` . I'll change it to ` UserLogsStyles`

That was @daniela-venuta  and @davidthai0387 part just make sure it's fine with them because maybe there was a reason behind it.

**Comment by nasaku898** [comment]
> While I was making those changes I noticed that error pop up but I was fairly sure I had fixed it because it all runs fine on my end? Should I just change it manually from Git? Because it looks fine on my end and the UserLogs component works fine on my end.
> image
> 

I think Git doesn't pick up file name change since the name is still the name. I'd recommend just modify something in that file and commit it

File: bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx [hunk 1]
```diff
@@ -1,21 +1,23 @@
-import { Button, Paper, Typography } from "@material-ui/core";
 import axios from "axios";
-import React, { useState, useEffect } from "react";
-import { BACKEND_URL } from "../../core/utils/config";
-import "./OrderBiling.css";
-import { Order, removeAllItem, removeItem } from "../../redux/actions/OrderListActions/orderListAction";
+import { useState, useEffect } from "react";
 import { connect } from "react-redux";
+
```
Comment (Manraj45) [inline bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx:4]: Remove Space

File: bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
+import { BACKEND_URL } from "../../../core/utils/config";
+import { Order, removeAllItem, removeItem } from "../../../redux/actions/OrderListActions/orderListAction";
+
```
Comment (Manraj45) [inline bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx:7]: Remove Space

File: bike-erp/src/components/OrderService/OrderComponent/ComponentView.tsx [hunk 3]
```diff
@@ -1,43 +1,30 @@
-import { CardMedia, Card, CardActions, Select, FormControl, InputLabel, MenuItem } from "@material-ui/core";
 import bike_logo from "../../../assets/images/login_bike_logo.png";
-import { Box,Typography } from "@material-ui/core";
-import "./ModelView.css";
```
Comment (Manraj45) [inline bike-erp/src/components/OrderService/OrderComponent/ComponentView.tsx:2]: Remove Space

File: bike-erp/src/components/PermissionManagement/PermissionDropdown.tsx [hunk 4]
```diff
@@ -1,9 +1,10 @@
+import { useState } from "react";
+
```
Comment (Manraj45) [inline bike-erp/src/components/PermissionManagement/PermissionDropdown.tsx:2]: Remove space plz

File: bike-erp/src/components/PermissionManagement/UserAccountsTable.tsx [hunk 5]
```diff
@@ -1,18 +1,24 @@
-import { Button, Table, TableBody, TableRow } from "@material-ui/core";
-import SaveIcon from "@material-ui/icons/Save";
 import axios from "axios";
 import { useEffect, useState } from "react";
+
```
Comment (Manraj45) [inline bike-erp/src/components/PermissionManagement/UserAccountsTable.tsx:3]: Remove space here too

File: bike-erp/src/index.tsx [hunk 6]
```diff
@@ -1,13 +1,14 @@
-import { ThemeProvider } from "@material-ui/core";
-import React from "react";
 import ReactDOM from "react-dom";
 import { Provider } from "react-redux";
+
```
Comment (Manraj45) [inline bike-erp/src/index.tsx:3]: Remove Space here

File: bike-erp/src/test/Inventory/Inventory.test.tsx [hunk 7]
```diff
@@ -1,8 +1,8 @@
 import Enzyme, { shallow } from "enzyme";
```
Comment (nasaku898) [inline bike-erp/src/test/Inventory/Inventory.test.tsx:1]: Could you remove the console.log on this test. I requested it last time, but it was not removed.

File: bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx (hunk 1 above)
Comment (emilie-martin) [reply bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx:4]: As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

File: bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx (hunk 2 above)
Comment (emilie-martin) [reply bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx:7]: As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

File: bike-erp/src/components/OrderService/OrderComponent/ComponentView.tsx (hunk 3 above)
Comment (emilie-martin) [reply bike-erp/src/components/OrderService/OrderComponent/ComponentView.tsx:2]: As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

File: bike-erp/src/components/PermissionManagement/PermissionDropdown.tsx (hunk 4 above)
Comment (emilie-martin) [reply bike-erp/src/components/PermissionManagement/PermissionDropdown.tsx:2]: As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

File: bike-erp/src/components/PermissionManagement/UserAccountsTable.tsx (hunk 5 above)
Comment (emilie-martin) [reply bike-erp/src/components/PermissionManagement/UserAccountsTable.tsx:3]: As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

File: bike-erp/src/index.tsx (hunk 6 above)
Comment (emilie-martin) [reply bike-erp/src/index.tsx:3]: As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

File: bike-erp/src/test/Inventory/Inventory.test.tsx (hunk 7 above)
Comment (emilie-martin) [reply bike-erp/src/test/Inventory/Inventory.test.tsx:2]: It has been done 👍 hadn't even noticed them. If I had, I would have removed them 👍

**Review by Manraj45 — APPROVED** [review]
Good job! Don't forget to squash merge

**Review by nasaku898 — APPROVED** [review]

**Pull request merged**